extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/NettyRecorder.java

```diff
@@ -25,7 +25,9 @@ public void run() {
                 DefaultChannelId.newInstance();
                 if (System.currentTimeMillis() - start > 1000) {
                     log.warn(
-                            "Localhost lookup took more than one second, you need to add a /etc/hosts entry to improve Quarkus startup time. See https://thoeni.io/post/macos-sierra-java/ for details.");
+                            "Localhost lookup took more than one second, you need to add your hostname ("
+                                    + System.getenv("HOSTNAME")
```

Comment:
The host name might or might not be in the env, and also the host name might not correspond to the actual network name of the network interface.  It might be better to say "You need a hosts entry for `localhost` which maps to `127.0.0.1` and `::1` in order to improve startup time" or "You need a hosts entry corresponding to IP address <whatever>" or something along those lines.

Reply:
My hosts file already had `localhost`. The error was gone only after I added `ggastald-laptop` in

Reply:
Fair 🙂 I didn't "request changes"!

Reply:
> It's the DNS name of the network interface that it used

Right, what I meant is that is what's returned when I type `hostname` in bash :smiley:

Reply:
Exactly.  So it's not exactly the right thing, and there might be multiple problematic interfaces besides.  We might want to change this check to run earlier at startup and have it iterate the `NetworkInterface` addresses trivially, with timing info.  Then when it's done we could actually list every IP/host name pair that the user could even cut & paste into the hosts file if they wanted to.